kolibri/plugins/learn/assets/src/constants.js

```diff
@@ -40,3 +43,59 @@ export const pageNameToModuleMap = {
   [PageNames.TOPICS_TOPIC]: 'topicsTree',
   [PageNames.TOPICS_TOPIC_SEARCH]: 'topicsTree',
 };
+
+const availablePaths = {};
+
+plugin_data.categories.map(key => {
+  const paths = key.split('.');
+  let path = '';
+  for (let path_segment of paths) {
+    path = path === '' ? path_segment : path + '.' + path_segment;
+    availablePaths[path] = true;
+  }
+});
+
+export const libraryCategories = {};
```

Comment:
I also hadn't realized we now had it in 3 different places, so I was inadvertently invoking the Rule of Three!